### PR TITLE
1.4 blog post.

### PR DESCRIPTION
**Pull request by nathanhammond**

### CODE REVIEW

File: docs/pages/blog/turbo-1-4-0.mdx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+- Docker
+- Create React App
+- React Native
+- Prisma
+- Remix
+- … and more!
+
+## Automatic environment variable inclusion
+
+One of the easiest mistakes to make when setting up Turborepo is accidentally omitting an important environment variable resulting in reusing a cache somewhere you didn't expect. Turborepo now has your back and will detect commonly used environment variables to help you avoid this problem.
```
Comment (tknickman) [inline docs/pages/blog/turbo-1-4-0.mdx:114]: Maybe worth listing the frameworks here that we auto detect?

File: docs/pages/blog/turbo-1-4-0.mdx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+date: 2022/07/28
+description: Turborepo 1.4 introduces the Turborepo Agent, has new examples, automatically hashes environment variables, and more!
+tag: web development
+ogImage: /images/blog/turbo-1-4-0/og-image.png
+---
+
+# Turborepo 1.4
+
+import { Authors } from '../../components/Authors'
+import Callout from '../../components/Callout'
```
Comment (tknickman) [inline docs/pages/blog/turbo-1-4-0.mdx:12]: Is this blog authors? Or contributors to the content? I always thought it was the latter based on the previous posts. 

I would argue _this_ is the correct way to do it though.

**Review by tknickman — APPROVED** [review]

File: docs/pages/blog/turbo-1-4-0.mdx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+- Svelte
+- Docker
+- Create React App
+- React Native
+- Prisma
+- Remix
+- … and more!
+
+## Automatic environment variable inclusion
```
Comment (jaredpalmer) [inline docs/pages/blog/turbo-1-4-0.mdx:112]: - Include links to documentation around env vars.
- List out the frameworks that are autodetected and articulate the strategy we use to do so. This impacts caching and so we need to add more information. Instruct folks that have listed these env vars in `dependsOn` to safely remove them

File: docs/pages/blog/turbo-1-4-0.mdx [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+---
+
+# Turborepo 1.4
+
+import { Authors } from '../../components/Authors'
+import Callout from '../../components/Callout'
+
+<div className="text-sm mt-2 text-center text-gray-500 dark:text-gray-400">Thursday, July 28th, 2022</div>
+
+<Authors authors={[ 'anthonyshewdev', 'nathanhammond', ]} />
```
Comment (jaredpalmer) [inline docs/pages/blog/turbo-1-4-0.mdx:16]:
```suggestion
<Authors authors={[ 'gsoltis', 'nathanhammond', 'tknickman' ]} />
```

File: docs/pages/blog/turbo-1-4-0.mdx [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+That interprocess communication uses gRPC and protobufs to give us versioning guarantees (both backwards- and forwards-compatible) and quick marshaling of data so that we don’t spend all of our time chatting.
+
+Unsupervised processes are wasteful and the majority of the code that was added was to make process handling as robust as possible.
+
+## New examples
+
+Over the past month we’ve invested a lot of time in [creating new examples for how to integrate Turborepo into your workflow](https://github.com/vercel/turborepo/tree/main/examples), featuring:
+
+- Svelte
```
Comment (jaredpalmer) [inline docs/pages/blog/turbo-1-4-0.mdx:104]: Should we linkify these?

File: docs/pages/blog/turbo-1-4-0.mdx (hunk 2 above)
Comment (nathanhammond) [reply docs/pages/blog/turbo-1-4-0.mdx:12]: It's been kinda unclear, but I'm going with post authors here. Contributors to the release have a spot at the bottom.

File: docs/pages/blog/turbo-1-4-0.mdx [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+- Properly rebuild packages that share a name prefix. ([#1538](https://github.com/vercel/turborepo/pull/1538))
+- Cache files with the correct file permissions. ([#1429](https://github.com/vercel/turborepo/pull/1429))
+
+## Community
+
+Since releasing [Turborepo v1.3 in June](/blog/turbo-1-3-0), we've seen incredible adoption and community growth:
+
+- [8.5k+ GitHub Stars](https://github.com/vercel/turborepo)
+- as many as 325k weekly NPM downloads
+- 8.1 years of compute time saved through Remote Caching on Vercel, saving 8 months per week
```
Comment (jaredpalmer) [inline docs/pages/blog/turbo-1-4-0.mdx:128]: It's actually 9.23 as of today. 10months last week

File: docs/pages/blog/turbo-1-4-0.mdx [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+- Allow both sides of git comparison. ([#1442](https://github.com/vercel/turborepo/pull/1442))
+- Properly rebuild packages that share a name prefix. ([#1538](https://github.com/vercel/turborepo/pull/1538))
+- Cache files with the correct file permissions. ([#1429](https://github.com/vercel/turborepo/pull/1429))
+
+## Community
+
+Since releasing [Turborepo v1.3 in June](/blog/turbo-1-3-0), we've seen incredible adoption and community growth:
+
+- [8.5k+ GitHub Stars](https://github.com/vercel/turborepo)
+- as many as 325k weekly NPM downloads
```
Comment (jaredpalmer) [inline docs/pages/blog/turbo-1-4-0.mdx:127]:
```suggestion
- 325k weekly NPM downloads, up 2x since late April
```

**Review by jaredpalmer — CHANGES_REQUESTED** [review]

File: docs/pages/blog/turbo-1-4-0.mdx [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+---
+
+# Turborepo 1.4
+
+import { Authors } from '../../components/Authors'
+import Callout from '../../components/Callout'
+
+<div className="text-sm mt-2 text-center text-gray-500 dark:text-gray-400">Thursday, July 28th, 2022</div>
+
+<Authors authors={[ 'gsoltis', 'nathanhammond', 'tknickman' ]} />
```
Comment (jaredpalmer) [inline docs/pages/blog/turbo-1-4-0.mdx:16]:
```suggestion
<Authors authors={[ 'gsoltis', 'nathanhammond', 'tknickman', 'anthonyshewdev' ]} />
```

File: docs/pages/blog/turbo-1-4-0.mdx (hunk 6 above)
Comment (nathanhammond) [reply docs/pages/blog/turbo-1-4-0.mdx:128]: This is scoped to remote caching so 8.1 is the correct number. https://vercel.com/solutions/turborepo

File: docs/pages/blog/turbo-1-4-0.mdx (hunk 5 above)
Comment (nathanhammond) [reply docs/pages/blog/turbo-1-4-0.mdx:104]: Done.

File: docs/pages/blog/turbo-1-4-0.mdx (hunk 1 above)
Comment (nathanhammond) [reply docs/pages/blog/turbo-1-4-0.mdx]: Enumerated.

**Pull request merged**